gax/src/test/java/com/google/api/gax/rpc/CancellationHelpers.java

```diff
@@ -48,7 +49,7 @@ public void run() {
                 try {
                   latch.await();
                   while (!resultFuture.cancel(true) && !resultFuture.isDone()) {
-                    Thread.sleep(1L);
+                    LockSupport.parkNanos(1000L);
```

Comment:
Why is this necessary?

Reply:
This switches sleep from 1ms to 1μs, to reduce flakiness of the tests.